treeopt/treeOptData/DoeResponce.csv

```diff
@@ -0,0 +1,20 @@
+1.768609252109100680e+02
```

Comment:
What is this? I have a hunch this does not belong into the source directory `treeopt`, but rather into `tests/data`?